contrib/build_compat_tables.py

```diff
@@ -61,20 +61,28 @@
     :width: 32
 """
 
-# Credit http://stackoverflow.com/questions/42195468/how-can-i-inspect-an-attribute-of-a-module-without-importing-it?noredirect=1#comment71553012_42195468
+# Credit http://stackoverflow.com/questions/42195468/
+#        how-can-i-inspect-an-attribute-of-a-module-without-importing-it?noredirect=1#comment71553012_42195468
+
```

Comment:
The long line doesn't need to be split here; splitting means the URL is no longer easily followed. If this is getting flagged by flake8, you can disable the lint check on this single line.

Reply:
I'll go ahead and fix that up...never disabled a lint for a single line before, sounds like a good skill to have. Thanks!

Reply:
Actually I did one better and used the "shortened" version of the SO link as per https://meta.stackexchange.com/questions/164194/is-there-a-way-to-shorten-stack-overflow-urls and tested it successfully as pointing to the same SO comment as the original link.